pulp_smash/tests/rpm/api_v2/test_upload_publish.py

```diff
@@ -149,6 +149,78 @@ def _get_pkglist(update):
     return out_pkglist
 
 
+class UploadDrpmTestCase(utils.BaseAPITestCase):
+    """Test whether one can upload and associate DRPMs."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Upload DRPM to a repo.
+
+        Steps:
+
+        1. Create a RPM repo.
+        2. Upload DRPM to repo.
+        3. Check if DRPM has been"uploaded.
+
+        https://github.com/PulpQE/pulp-smash/issues/336
```

Comment:
Writing something like:

    This test case targets `Pulp Smash #336`_.

...will save future readers a bit of time. Doing this means that the user can know something about the linked-to information without having to context-switch and move to a web browser.